aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -3,15 +3,335 @@ title: Create a .NET Core gRPC client and server in ASP.NET Core
 author: juntaoluo
 description: This tutorial shows how to create a gRPC Service and gRPC client on ASP.NET Core. Learn how to create a gRPC Service project, edit a proto file, and add a duplex streaming call.
 ms.author: johluo
-ms.date: 08/20/2021
+ms.date: 09/13/2021
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/grpc/grpc-start
 ---
 # Tutorial: Create a gRPC client and server in ASP.NET Core
 
 By [John Luo](https://github.com/juntaoluo)
 
-::: moniker range=">= aspnetcore-5.0"
+::: moniker range=">= aspnetcore-6.0"
```

Comment:
Next doc you update for V6, do a moniker update PR to create the V6 section and S&M. Then to update the doc, work on the V6 section so you have a good diff file to see what's changed. When you add the V6 moniker section to the old file, GitHub can't create a useful diff.

Reply:
Great idea!  Thanks, I will do do moniker updates that way from now on.